### PR TITLE
[chore] remove polygon mumbai references

### PR DESCRIPTION
**Pull request by kasparkallas**
Why?
Mumbai is not producing blocks anymore. Any reference to it is a unnecessary distraction.

TODO:
* Didn't deprecate from the token list yet.

### CODE REVIEW

File: sdk-redux-examples/sdk-redux-nextjs-typescript/redux/chainIds.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export const chainIds = [
     // 100, // XDAI //TODO(KK): No infura support
     137, // MATIC
-    80001, // MUMBAI
+    43113 , // OP Sepolia
```
Comment (kasparkallas) [inline sdk-redux-examples/sdk-redux-nextjs-typescript/redux/chainIds.ts:4]: Oops, Fuji actually

Comment (hellwolf) [reply sdk-redux-examples/sdk-redux-nextjs-typescript/redux/chainIds.ts:4]: okay,

File: packages/automation-contracts/autowrap/README.md [hunk 2]
```diff
@@ -58,7 +57,7 @@ npx hardhat addStrategy --manager <manager_address> --strategy <strategy_address
 #### Testnets
 |          | Manager                                                                                                                              | WrapStrategy                                                                                                                         |
 |----------|--------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
-| Mumbai   | [0x3eAB3c6207F488E475b7955B631B564F0E6317B9](https://mumbai.polygonscan.com/address/0x3eAB3c6207F488E475b7955B631B564F0E6317B9#code) | [0x544728AFDBeEafBeC9e1329031788edb53017bC4](https://mumbai.polygonscan.com/address/0x544728AFDBeEafBeC9e1329031788edb53017bC4#code) |
+| OP Sepolia   | [0xe567b32C10B0dB72d9490eB1B9A409C5ADed192C](https://sepolia-optimism.etherscan.io/address/0xe567b32C10B0dB72d9490eB1B9A409C5ADed192C) | [0xf232f1fd34CE12e24F4391865c2D6E374D2C34d9](https://sepolia-optimism.etherscan.io/address/0xf232f1fd34CE12e24F4391865c2D6E374D2C34d9) |
```
Comment (hellwolf) [inline packages/automation-contracts/autowrap/README.md:60]: Is it Fuji also?

**Review by hellwolf — APPROVED** [review]

File: packages/automation-contracts/autowrap/README.md (hunk 2 above)
Comment (kasparkallas) [reply packages/automation-contracts/autowrap/README.md:60]: Nop, OP Sepolia

**Pull request merged**